src/main/java/com/github/gokolo/personaldashboard/dao/AddressDAOImpl.java

```diff
@@ -55,6 +57,24 @@ public AddressDTO save(final AddressDTO address) {
     }
 
     @Override
+    public List<AddressDTO> findAll() {
+        Statement statement;
+        ResultSet resultSet;
+        List<AddressDTO> addressCollection = new ArrayList<>();
+        try {
+            statement = conn.createStatement();
+            resultSet = statement.executeQuery("SELECT * FROM address");
+            while (resultSet.next()) {
+                addressCollection.add(convert(resultSet));
+            }
+        } catch (final SQLException e) {
+            // TODO Auto-generated catch block
```

Comment:
Be mindful of what is auto-generated by the IDE you're using.
Don't leave unnecessary stuff in your code.
Also remember to always log the exception when you are catching one.
`e.printStackTrace()` calls should generally be avoided, we don't want to pollute the standard output. This is why we have logging.

No need to fix this now, this is just a reminder.